Cargo.toml

```
@@ -20,16 +20,19 @@ exclude = [
 travis-ci = { repository = "dalek-cryptography/x25519-dalek", branch = "master"}
 
 [dependencies.curve25519-dalek]
-version = "^0.19"
+version = "^1.0.0-pre.1"
```

```suggestion
version = "1"
```
now that the stable version is released